fft_g1.go

```diff
@@ -12,13 +12,13 @@ func (fs *FFTSettings) simpleFTG1(vals []G1, valsOffset uint64, valsStride uint6
 	for i := uint64(0); i < l; i++ {
 		jv := &vals[valsOffset]
 		r := &rootsOfUnity[0]
-		mulG1(&v, jv, r)
+		mulG1(&v, jv, norm(r))
```

Comment:
Hmm, repeating `norm` for every root of unity seems like something that could be precomputed.